css/properties/-moz-user-focus.json

```diff
@@ -0,0 +1,57 @@
+{
+  "css": {
+    "properties": {
+      "-moz-user-focus": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/-moz-user-focus",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": true
```

Comment:
This should have `"partial_implementation": "true"` with `"note": "While the CSS property is parsed and accepted, it does not have any effect."`

Reply:
I'm a little skeptical about the validity of this information -- it is not mentioned in the MDN document, and probably would count as deprecated if that is the case.  Is this possibly a comment meant for user-modify.json instead?

Reply:
This should be marked as resolved (see&nbsp;
https://github.com/mdn/browser-compat-data/pull/2868#discussion_r22145464)9